terms.html

```diff
@@ -5,12 +5,20 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made, by a specific <a>entity</a>, about a <a>resource</a>.
+This <a>resource</a> is called the <a>subject</a> of the <a>claim</a>.
+This digital representation usually consists of an (optional) identifier (an [IRI](https://datatracker.ietf.org/doc/html/rfc3987))
```

Comment:
The specification uses the term "URL", not "IRI".

Reply:
From what I read, IRI goes with RDF (subject) identifiers, and URLs are a subclass thereof. Is there a particular reason for having URLs here rather than IRIs (apart, perhaps, from the fact that they are specified as such in `index.html`)?
I personally do not have a preference, so I'm happy to go by whatever it needs to be

Reply:
ok